packages/gestalt/src/Masonry.js

```diff
@@ -205,6 +205,7 @@ export default class Masonry<T: {}> extends React.Component<
     const measurementStore: Cache<T, *> =
       props.measurementStore || Masonry.createMeasurementStore();
 
+    // eslint-disable-next-line react/state-in-constructor
```

Comment:
Going to leave this for now since Airbnb will also enable it moving forward (we want to stay as closely as possible to their config)

https://github.com/airbnb/javascript/blob/2e3adc98c784be7fa63c7a05271f7aa86c9d6bc7/packages/eslint-config-airbnb/rules/react.js#L484

```
// TODO: set to "never" once babel-preset-airbnb supports public class fields
```